ui/src/pages/Logs/Logs.tsx

```diff
@@ -116,11 +117,33 @@ export default function Logs() {
         {}
       );
       setContainerIconColor(updatedContainerIconColor);
+
+      setElapsedTimeInMinutes(0);
     } catch (err) {
       console.error(err);
     }
   };
 
+  useEffect(() => {
```

Comment:
I like the idea of using a fuzzy timestamp (x minutes ago) which you've done here! I was imagining just displaying a timestamp in the date time format, but this is nice.

I think displaying both could be even better, what do you think? Something like:

```
Last refreshed YYYY-MM-DDTHH:mm:ss.sss ( 3 minutes ago )
```

The timestamp would be grabbed in the refreshLogs function